lib/datadog/tracing/span_event.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'time'
+
+module Datadog
+  module Tracing
+    # SpanEvent represents an annotation on a span.
+    # @public_api
+    class SpanEvent
+      # @!attribute [r] name
+      #   @return [Integer]
+      attr_reader :name
```

Comment:
It is uncommon for `name` to be an integer, can the docstring be expanded to provide more information on this? Is the integer coming from some other system and therefore is expected to mean something specific?

Reply:
Oh that's a miss on my part. It should be a string